wiki/Help_Centre/Bot_Account/en.md

```diff
@@ -0,0 +1,53 @@
+---
+needs_cleanup: false
+---
+
+# Bot Account criteria
```

Comment:
```suggestion
# Bot Account criteria
```
if the article does not require any clean-up, there is no need to specifically mention it here, afaik